### PR TITLE
Need visual density option in grouped_checkbox

### PR DESCRIPTION
**Pull request by nicks258**
## Connection with issue(s)

Close: #1283 

## Solution description
Added Visual density parameter and passed through `FormBuilderCheckboxGroup`

## To Do

- [x] Read [contributing guide](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/CONTRIBUTING.md)
- [x] Check the original issue to confirm it is fully satisfied
- [x] Add solution description to help guide reviewers
- [ ] Add unit test to verify new or fixed behaviour
- [ ] If apply, add documentation to code properties and package readme

### CODE REVIEW

File: lib/src/fields/form_builder_checkbox_group.dart [hunk 1]
```diff
@@ -25,7 +26,7 @@ class FormBuilderCheckboxGroup<T> extends FormBuilderFieldDecoration<List<T>> {
   final OptionsOrientation orientation;
 
   /// Creates a list of Checkboxes for selecting multiple options
-  FormBuilderCheckboxGroup({
+  FormBuilderCheckboxGroup(this.visualDensity, {
```
Comment (deandreamatias) [inline lib/src/fields/form_builder_checkbox_group.dart:29]: Move inside {} to convert this to parameter named

**Review by deandreamatias — CHANGES_REQUESTED** [review]
Hi! Thanks for contribution.
Please format code

File: lib/src/fields/form_builder_checkbox_group.dart (hunk 1 above)
Comment (Sumit258000) [reply lib/src/fields/form_builder_checkbox_group.dart]: My bad! Done Please check now.

Comment (nicks258) [reply lib/src/fields/form_builder_checkbox_group.dart]: My Bad! Done. Please check.

**Review by Sumit258000 — APPROVED** [review]

File: lib/src/fields/form_builder_checkbox_group.dart (hunk 1 above)
Comment (Sumit258000) [reply lib/src/fields/form_builder_checkbox_group.dart]: Did you push this changes to version 9.1.1? @deandreamatias

**Comment by nicks258** [comment]
I am not able to see my commits in main branch. Can you please check. @deandreamatias

**Comment by deandreamatias** [comment]
@nicks258   Please format code

**Comment by deandreamatias** [comment]
@nicks258 you can see in the [failed pipeline](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/actions/runs/6258205997/job/16994473070?pr=1284), that need format two files:
Screenshot 2023-09-21 at 11 28 59

To solve this, go to your fork project, execute `dart format --set-exit-if-changed .` and commit the changes

**Comment by nicks258** [comment]
Thanks for your message. I formatted my code and pushed.

**Pull request merged**